scripts/vim-patch.sh

```diff
@@ -565,13 +576,13 @@ show_vimpatches() {
     runtime_commits[$commit]=1
   done
 
-  list_missing_vimpatches 1 "$@" | while read -r vim_commit; do
+  while read -r vim_commit; do
```

Comment:
We're not making any changes inside the loop that need to be visible outside of it, so SC2030 is technically a false positive here.